src/sass/myservice-widgets.scss

```diff
@@ -271,4 +271,10 @@ table.table-claims {
 
 .healthcard-steps p {
     margin: 10px 0 10px 0;
+}
+
+#card-tpi {
```

Comment:
this Id should be nested within #healthcard-container and position from the top should be a value that makes it line up nicely with #card-number